### DISCUSSION TITLE
Issue with connecting to OHIF from MONAILabel server

### DISCUSSION THREAD

**Discussion by neerajamaha** [Project-MONAI/MONAILabel#1492] [Q&A]
Hello,

I am running a segmentation model using the MONAI Label client with my studies in the Orthanc server. I have launched the server and am able to pull images and run inferences in 3D Slicer. However, when I try to run through the same workflow using OHIF, I am getting this error.

![image](https://github.com/Project-MONAI/MONAILabel/assets/98112236/2937c0ed-ef76-474f-b521-0d462b2e9c6b)

We have tried running the server and client on the same machine and well as on separate machines and both methods produce the same error. We have also tried updating the Orthanc docker image but are running into the same issues. 

I am not sure how to view the MONAILabel server logs, so if I could get some guidance on how to get that, as well as on the issues I'm having with OHIF, it would be appreciated.

Thank you!

**Comment by SachidanandAlle** [comment]
How r u using monailabel? From GitHub source?

In that case you have to build OHIF.. (mentioned in the instructions I believe) or else install monailabel via pip

Also you can share your command to run monailabel server for debugging

**Reply by diazandr3s** [reply]
Thanks for opening this discussion, @neerajamaha.
As @SachidanandAlle suggested, please build OHIF following these instructions: https://github.com/Project-MONAI/MONAILabel/tree/main/plugins/ohif#installing-ohif

**Reply by neerajamaha** [reply]
Hi, thank you both for the help. I have installed monailabel using pip. So there should be a pre-built OHIF also installed, correct?

Here is the command I am using to run the monailabel server:
monailabel start_server --app radiology_optimized-dynunet_pretrainedTS-TCV_withclicks --studies http://10.200.16.30:8042/dicom-web --conf models deepedit --conf use_pretrained_model false
